index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Developer Hackerton Naija 2021</title>
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="stylesheet" href="assets/css/mobile.css" media="screen and (max-width: 768px)">
+    <link rel="stylesheet" href="assets/css/desktop.css" media="screen and (min-width: 768px)">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="bar-section">
+        <nav class="top-header">
+            <div class="top-wrapper">
+                <ul class="link-rx">
+                    <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa fa-facebook-square"></i></a></li>
+                </ul>
+                <ul class="link-info">
+                    <li>English</li>
+                    <li>Sign Up</li>
+                    <li>Login</li>
+                </ul>
+            </div>
+        </nav>
+        <nav class="nav-bar">
+            <div class="logo">
+                <a href="index.html#welcome" tabindex="1">
+                    <img src="./assets/images/conference_logo.png" alt="Conference Organizers Logo" class="conference-logo logo-effect"/>
```

Comment:
I really like this part. Good semantic tag application. Keep up the good work. :muscle: